src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -206,6 +209,11 @@ public function shutdown()
             $isFailingAtShutdown = !$configuration->tolerates($this->deprecationGroups);
             $this->displayDeprecations($groups, $configuration, $isFailingAtShutdown);
 
+            if ($configuration->isGeneratingBaseline()) {
+                $configuration->writeBaseline();
+                echo sprintf("Baseline written. To use the baseline set SYMFONY_DEPRECATIONS_HELPER='baselineFile=%s'\n", urlencode($configuration->getBaselineFile()));
```

Comment:
let's remove this echo: if ppl triggered a "generate", they know how to turn the flag to false

Reply:
Well it makes end-to-end tests more difficult. But I've done this. See changes to src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/baseline.phpt for an example of how this makes things a bit tricker. At least we're still asserting that DeprecationErrorHandler doesn't emit any deprecation messages I guess.